src/test/java/formflow/library/utilities/FormScreen.java

```diff
@@ -29,8 +29,7 @@ public boolean hasInputError() {
   }
 
   public boolean hasInputError(String inputName) {
-    Element element = html.select("input[name='%s'] ~ p.text--error".formatted(inputName))
-        .first();
+    Element element = html.select("[id^=\"%s-error-message\"]".formatted(inputName)).first();
```

Comment:
Not really sure why the `hasInputError` was checking something differently than the `getInputError`, but changed this to match that. I had a case where there was an error, and `getInputError` would fetch it correctly, but `hasInputError` said there wasn't an error.